plugin/core/protocol.py

```diff
@@ -70,7 +83,7 @@ class SignatureHelpTriggerKind:
 
 
 CodeLens = TypedDict('CodeLens', {
-    'range': Dict[str, Any],
+    'range': Range_T,
```

Comment:
Maybe `RangeLsp` instead? The `T` is a common naming convention when dealing with generics but it's not the case here.

Reply:
done 5f1116f6373c899615e556fdaed08ffd03e3e2f8